Dockerfile

```diff
@@ -76,8 +78,8 @@ RUN cd ${TEMP} \
 && rm -rf ${TEMP} \
 \
 # Add group and user
-&& groupadd -g 1000 nexus \
-&& adduser -u 997 -d ${IQ_HOME} -c "Nexus IQ user" -g nexus -s /bin/false -r nexus \
+&& groupadd -g ${GID} nexus \
+&& adduser -u ${UID} -d ${IQ_HOME} -c "Nexus IQ user" -g nexus -s /bin/false -r nexus \
```

Comment:
There's also `Dockerfile.rh` which we should make the same changes in I think

Reply:
> There's also `Dockerfile.rh` which we should make the same changes in I think

Updated in 7598e14941069b198c010f49a978cf4723e43bc8.